src/config/dynamic-theme-fixes.config

```diff
@@ -3364,6 +3364,14 @@ INVERT
 .mw-mmv-filepage-buttons .mw-mmv-view-expanded .mw-ui-icon::before
 .mw-mmv-filepage-buttons .mw-mmv-view-config .mw-ui-icon::before
 .licensetpl td[style^="width"]
+#p-personal .mw-echo-notifications-badge
+.mw-ui-icon::before
+.mw-kartographer-map
+.mw-kartographer-mapDialog-map
+.oo-ui-iconElement-icon
+.oo-ui-indicatorElement-indicator
+.mw-echo-notifications-badge
+img[alt="audio speaker icon"]
```

Comment:
I'm not sure if all these elements will actually apply something onto those websites. Certain things like `.mw-kartographer-map` would need to be checked because I don't think every website will have maps to be displayed.

Reply:
After recheck, I think Kartographer isn't needed outside Wikipedia (as originally applied by #8371)

I removed Kartographer lines and left the remaining (they are used by notification badge and editing toolbar per screenshots)